docs/workshop/index.md

```diff
@@ -11,15 +11,15 @@ In this 5-day training course, we will provide an introduction to the following
 
 1. **Login to PrairieLearn** at [https://prairielearn.engr.illinois.edu/pl/](https://prairielearn.engr.illinois.edu/pl/) (in case you have never done this before). This is necessary so that we can give you access to PrairieLearn before the workshop starts.
 
-2. **Request course access.** We are processing all the access requests using the PrairieLearn Slack [`#request-course`](https://go.illinois.edu/joinplslack) channel. Check the options below for instructions.
+2. **Request course access.** We are processing all the access requests using the PrairieLearn Slack [`#request-course`](https://go.illinois.edu/PrairieLearnSlack) channel. Check the options below for instructions.
```

Comment:
I know that we're not using this method for course creation anymore. Should we remove this outdated information from the docs?

Reply:
These docs are specifically for a workshop that I don't think we're running anymore. I'd be inclined to either leave them untouched (with the exception of just fixing this link) or slap a big "this page is outdated" warning on the top. Thoughts? cc @mfsilva22, I think you authored most of these?